esx_service/cli/vmdkops_admin.py

```diff
@@ -916,10 +916,7 @@ def generate_tenant_ls_rows(tenant_list):
         uuid = tenant.id
         name = tenant.name
         description = tenant.description
-        if not tenant.default_datastore_url or tenant.default_datastore_url == auth_data_const.DEFAULT_DS_URL:
-            default_datastore = ""
-        else:
-            default_datastore = vmdk_utils.get_datastore_name(tenant.default_datastore_url)
+        default_datastore = vmdk_utils.get_datastore_name(tenant.default_datastore_url)
```

Comment:
a) I think for xESX, the configuration we support assumes the datastore is shared. Am I miss anything here?
b) I don't think so, "VM_DS" is a special datastore tag,  and I don't think it should map for each VM in the tenant since those VM may live on different datastores.